FAQ: How can i speed up (#1566)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

### Does Wasabi support sending RBF?

All _send_ transactions signal RBF by default.

> A pending transaction may be bumped with RBF by using the [_Speed Up Transaction_ tool](/FAQ/FAQ-UseWasabi.md#how-can-i-speed-up-a-pending-transaction)

Reviewer: ```suggestion
A pending transaction may be bumped (RBF) by using the [_Speed Up Transaction_ feature](/FAQ/FAQ-UseWasabi.md#how-can-i-speed-up-a-pending-transaction)
```

These sats are precious, so don't overpay on fees!
:::

### How can I speed up a pending transaction?

A pending (unconfirmed) transaction can be speeded up by using the _Speed Up Transaction_ feature by right-clicking on the transaction in the history.

> The transaction will be speeded up by sending a new transaction with a higher fee rate.

Author: ```suggestion
The transaction will be speeded up by sending a new transaction with a higher fee rate.
```
to keep it simple, this is true for both RBF and CPFP

To speed up a transaction:

- Right click on the pending transaction in the wallet history and click _Speed Up Transaction_.

![Speed Up History](/SpeedUpHistory.png "Speed Up History")

- Confirm you want to pay the additional fee for speeding it up.

![Speed Up Confirm](/SpeedUpConfirm.png "Speed Up Confirm")

- Enter your Password to send the new transaction.
- Transaction succesfully speeded up!

> Read more [here](/using-wasabi/Send.md#speed-up-or-cancel-transaction).

Reviewer: This link here is currently broken.

@MarnixCroes I know that this will be fixed once #1567 is merged, but this is a bad practice to merge PRs with broken links or images.
You could have simply added this line in the follow up PR.

These sats are precious, so don't overpay on fees!
:::

### How can I speed up a pending transaction?

A pending (unconfirmed) transaction can be speeded up by using the _Speed Up Transaction_ feature by right-clicking on the transaction in the history.
It will replace the current transaction with one paying a higher fee rate.

<details>

> <summary>Click here to expand for a step-by-step instruction guide:</summary>

Reviewer: ```suggestion
<summary>Click here to expand for a step-by-step instruction guide:</summary>

```

These sats are precious, so don't overpay on fees!
:::

### How can I speed up a pending transaction?

> A pending (unconfirmed) transaction can be speeded up by using the _Speed Up Transaction_ tool by right-clicking on the transaction in the history.

Reviewer: ```suggestion
A pending (unconfirmed) transaction can be speeded up by using the _Speed Up Transaction_ feature by right-clicking on the transaction in the history.
```

> It will replace the current transaction with one paying a higher fee rate.

Reviewer: Here we can mention when RBF or CPFP is used.

<details>

> <summary>Click here to expand for a step-by-step instruction:</summary>

Reviewer: ```suggestion
<summary>Click here to expand for a step-by-step instruction guide:</summary>
```

> <br>

Reviewer: ```suggestion

```

### How can I bump the transaction fee with child pays for parent (CPFP)?

> Since Wasabi version 2.0.4 this FAQ is now obsolete because of the new [_Speed Up Transaction_ tool](/FAQ/FAQ-UseWasabi.md#how-can-i-speed-up-a-pending-transaction) that can be used.

Reviewer: ```suggestion
Since Wasabi version [2.0.4](https://github.com/zkSNACKs/WalletWasabi/releases/tag/v2.0.4) this FAQ is now obsolete because of the new [_Speed Up Transaction_ feature](/FAQ/FAQ-UseWasabi.md#how-can-i-speed-up-a-pending-transaction) that can be used.
```
